modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets.adoc_{context}"]
+= Create a Secret object by using the web console
```

Comment:
Our procedures should start with a gerund ("Creating").

I'd also recommend just saying "secret" instead of "Secret object", to be consistent with the other procedures

So: 

```
= Creating a secret by using the web console
```